src/session.rs

```diff
@@ -934,6 +934,14 @@ impl Session {
                 }
             }
             Command::ResizeFrame(fw, fh) => {
+                if fw == 0 || fh == 0 {
+                    self.message(
+                        "can not set frame dimension to `0`",
+                        MessageType::Warning,
+                    );
```

Comment:
Can you change this to "Error: cannot set frame dimension to `0`" and make it an `Error` message? I think that's more appropriate, since we're refusing to process the command.